src/ir_Kelvinator.cpp

```diff
@@ -184,13 +172,11 @@ uint8_t IRKelvinatorAC::calcBlockChecksum(const uint8_t *block,
 }
 
 /// Calculate the checksum for the internal state.
-/// @param[in] length Length of the internal state to checksum.
-void IRKelvinatorAC::checksum(const uint16_t length) {
+/// @param[in] length Deprecated
+void IRKelvinatorAC::checksum(const uint16_t /*length*/) {
```

Comment:
If this parameter is _truly_ no longer used, rather than mark it Deprecated, just remove it.
`IRKelvinatorAC::checksum()` should be a private (vs. a public function) so we can do with it as we need, as no user is going to be able to call it.